tests/submap_load_test.cpp

```diff
@@ -0,0 +1,1340 @@
+#include "catch/catch.hpp"
+#include "submap.h"
+
+#include <istream>
+
+#include "game.h"
+#include "make_static.h"
+#include "trap.h"
+#include "vehicle.h"
+
+static const point corner_ne = point_zero;
```

Comment:
If you want a 0/0 alias you can declare it as a const reference to point_zero.
```suggestion
static const &point corner_ne = point_zero;
```